asreviewcontrib/semantic_clustering/main.py

```diff
@@ -26,12 +26,18 @@ def execute(self, argv):
             version=f"{self.extension_name}: {self.version}", argv=argv)
 
         if args.filepath:
-            data = ASReviewData.from_file(args.filepath.name)
-            SemanticClustering(data, args.output)
+            data = ASReviewData.from_file(args.filepath)
+            SemanticClustering(
+                data,
+                args.output,
+                transformer=args.transformer)
 
         elif args.testfile:
             data = ASReviewData.from_file("https://raw.githubusercontent.com/asreview/systematic-review-datasets/master/datasets/van_de_Schoot_2017/output/van_de_Schoot_2017.csv")  # noqa: E501
```

Comment:
See comment above about `read_data`. Hardcoding URLs is very tricky (for example, we want to rename the `systematic-review-datasets` repo in the future). You can use the solution proposed above, or see this https://asreview.readthedocs.io/en/latest/API/generated/asreview.datasets.DatasetManager.html#asreview.datasets.DatasetManager